### ISSUE TITLE
.watsonrc not ignoring node_modules for a node project

### ISSUE THREAD

**Issue by sameersegal** [nhmood/watson-ruby#123]
Hey, great idea and project! This is something we hope to definitely use as part of our dev process. 

I am having trouble ignoring the node_modules dir at the root level. Due to this reason it runs really slow and fills up the screen with insane number of [o] node_modules/*_/_

```
# watson rc
# watson - inline issue manager
# [goosecode] labs


# Directories
[dirs]
./


# Tags
[tags]
fix
review
todo


# Ignores
[ignore]
.git
*.swp
node*
node_modules
node_modules/


[github_api]



[github_repo]


```

As you can see I tried various combinations `node*`, `node_modules/` and `node_modules`

Best, Sameer

**Comment by sameersegal** [comment]
Even the command line doesn't seem to work `watson --ignore node_modules` doesn't seem to work. Is there some issue with the `_` (underscore)

**Comment by nhmood** [comment]
There was an issue with the config file ignoring that was fixed with b34ec090403983775fda01b5ac259994ee7e1b42
Give that a try and let me know if that doesn't work.

But you do bring up a good point, the command line --ignore does not seem to be working with directories, I'll take a look.

**Comment by sameersegal** [comment]
Thanks for your quick response. Just wanted to check if you are planing to push an update soon. I saw that you have bumped up the version number

**Comment by nhmood** [comment]
I should have a fix for this by later today.

The version number bump came from following [Semantic Versioning](http://semver.org/) 
7da363704d6900438135b3591a353254d6f30222 added the -s/--show command line option (which I figured was a feature addition) so I bumped the minor # of the version.

If this is the incorrect convention please let me know so I can version properly!

**Comment by sameersegal** [comment]
I did `gem update watson` and it said that it is installing watson@1.1.1 which I assume is the latest.

I continue to see [o] node_modules/*_/_ entries

Then I tried - are you updating the version of your script?

```
$ watson --version
watson v1.0
Copyright (c) 2012-2013 goosecode labs
Licensed under MIT, see LICENSE for details

Written by nhmood, see <http://goosecode.com/projects/watson>
```

**Comment by nhmood** [comment]
@sameersegal yes, I wasn't updating the version in the help menu, this was fixed in https://github.com/nhmood/watson-ruby/commit/9a0f1af94e7b06ff62d6643d1fc8251c1d3e788c

The original issue with ignoring directories should be resolved in 5c2843a200b3933a7669f3e4a4d72ae1a7302b5a

I tested and it seems to be working so I'll close the issue, but if you are still having problems let me know and I'll reopen it

**Comment by sameersegal** [comment]
Hey, thanks for the quick turn around. Some thing seems to be working and something seems to be breaking:  I am using the same config as above and below is my directory structure. Now `node_modules` is getting ignored but other folders like apps are too.

```
$ ls -ltr
total 56
-rw-r--r--   1 S  staff   422 Feb 10  2013 README.md
drwxr-xr-x   4 S  staff   136 Feb 10  2013 views
drwxr-xr-x   5 S  staff   170 Feb 10  2013 public
drwxr-xr-x   7 S  staff   238 Mar  8  2013 bin
-rw-r--r--   1 S  staff  4738 Apr 11  2013 dump.rdb
drwxr-xr-x   5 S  staff   170 May 23  2013 docs
-rw-r--r--   1 S  staff   170 Jun  5 19:17 developer.md
drwxr-xr-x  31 S  staff  1054 Aug  5 16:35 node_modules
-rw-r--r--   1 S  staff   807 Aug 30 11:47 package.json
drwxr-xr-x  10 S  staff   340 Oct  2 15:25 apps
-rw-r--r--   1 S  staff  1640 Oct  2 15:26 server.js
-rw-r--r--   1 S  staff  1609 Oct  2 15:50 config.coffee
drwxr-xr-x   7 S  staff   238 Oct  2 15:54 test
drwxr-xr-x  11 S  staff   374 Oct  8 21:21 tmp
```

```
------------------------------
watson - inline issue manager

Run in: /Users/Sameer/Documents/Code/MotherShip
Run @ Wed Nov 27 08:14:16 2013
------------------------------

[ o ] public/stylesheets/style.css
[ o ] test/_helpers.js
[ o ] test/mocha.opts
[ o ] test/test_db.sql
[ o ] test/test_db_helpers.coffee
[ o ] test/spec/app/app.spec.coffee
.
.
.
```

When I remove the `./` under `[dirs]` and add `apps` I start to see my issues in the `apps` folder. However when I add `./` and `apps` I see the above incorrect output.

**Comment by nhmood** [comment]
@sameersegal I think I finally found the problem but I'd prefer if you could run it in your environment to verify (since I keep thinking I fix it but it fails in your env!)

I have built a new gem that can be downloaded from http://goosecode.com/watson/watson-ruby-sameersegal.gem
(md5sum: ef51032abc9ed3cfbe925a13bf3aabda)

If you prefer to build the gem from source let me know and I can create a branch in this repo for you to checkout and build

Fixed in repo with 2686c30f43beb0a41bb1a64630f9aff5931b67da

**Comment by sameersegal** [comment]
Perfect - the updated gem works well! Thank you so much
